From 1de6f8b28ff551fc361f0a00ba2f879074721a58 Mon Sep 17 00:00:00 2001
From: Fernando Reyna <fernando.m.reyna.diaz@intel.com>
Date: Wed, 17 Feb 2016 12:25:46 -0600
Subject: [PATCH 02/10] OV2680: Changed api_device_id.

CHT-T3 MRD is looking for OVTI2680 but the api_device_id
is using XXOV2685 so it was changed to work on MRD.

Change-Id: I865057435ad1799c9d6bdeac7cd8bdef3b807cf1
Original-Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26178
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26488
Signed-off-by: Fernando Reyna <fernando.m.reyna.diaz@intel.com>
Signed-off-by: Marco Salgado <marco.antonio.salgado.martinez@intel.com>
Signed-off-by: Lopez Esparza, Ricardo <ricardo.lopez.esparza@intel.com>
Reviewed-on: https://android.intel.com:443/471770
Reviewed-on: https://android.intel.com:443/487148
---
 drivers/media/i2c/atomisp/ov2680.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/atomisp/ov2680.c b/drivers/media/i2c/atomisp/ov2680.c
index 47db8df..021a31a 100644
--- a/drivers/media/i2c/atomisp/ov2680.c
+++ b/drivers/media/i2c/atomisp/ov2680.c
@@ -1644,6 +1644,7 @@ out_free:
 
 static struct acpi_device_id ov2680_acpi_match[] = {
 	{"XXOV2680"},
+	{"OVTI2680"},
 	{},
 };
 MODULE_DEVICE_TABLE(acpi, ov2680_acpi_match);
-- 
1.9.1

